rocky/rocky/views/ooi_view.py

```diff
@@ -181,12 +181,16 @@ def get_form_kwargs(self):
     def form_valid(self, form):
         # Transform into OOI
         try:
+            end_valid_time = form.cleaned_data.pop("end_valid_time", None)
+            if end_valid_time is not None:
+                end_valid_time = end_valid_time.astimezone(timezone.utc)
```

Comment:
```suggestion
                end_valid_time = end_valid_time.replace(tz=timezone.utc)
```

Although this seems to be dead code that is not used because we don't have a form field?

Reply:
We will, once we figure out how to trigger the garbage collection once an OOI times out.

Reply:
> Although this seems to be dead code that is not used because we don't have a form field?

Actually the Single MuteFindingView passes through here, eventually... 🍝